docs/schema/reference.md

```diff
@@ -332,19 +332,19 @@ In most circumstances, the `payer` identifier will match that of the `buyer`, an
 
 See [milestone](#milestone) reference below.
 
-The implementation milestones should be updated to reflect when they are met. 
+The implementation milestones should be updated to reflect when they are met.
 
 #### Documents
 
 Documents related to contract implementation are stored here. This can include subcontracts.
 
-See [document](#document) reference below. 
+See [document](#document) reference below.
 
 ### Amendment
 
 A release may amend values from a previous release. Whilst the release & record model of OCDS offers the opportunity to keep a full versioned history of changes, in many cases it is important for changes to a tender, award or contract to be explicitly declared.
 
-The amendment array in a tender, award or contract block provides the ability to detail the amendments that have taken place with dates, rationale and free-text descriptions of the change, as well as to point to the releases that contain information from before and after the amendment.
+The `amendments` array in a `tender`, `Award` or `Contract` object provides the ability to detail the amendments that have taken place with dates, rationale and free-text descriptions of the change, as well as to point to the releases that contain information from before and after the amendment. Each amendment is detailed using the `Amendment` subschema.
```

Comment:
```suggestion
The `amendments` array in a `tender`, `Award` or `Contract` object provides the ability to detail the amendments that have taken place with dates, rationale and free-text descriptions of the change, as well as to point to the releases that contain information from before and after the amendment.
```

Reply:
again I think this is useful to link the subschema that will populate the array